applications/opentelemetry-operator/0.93.0/helmrelease/opentelemetry.yaml

```
@@ -0,0 +1,38 @@
+---
+apiVersion: source.toolkit.fluxcd.io/v1
+kind: OCIRepository
+metadata:
+  name: ghcr.io-opentelemetry-operator
+  namespace: ${workspaceNamespace}
+spec:
+  url: oci://ghcr.io/open-telemetry/opentelemetry-helm-charts/opentelemetry-operator
+  ref:
+    tag: 0.93.0
+  interval: 10m
+  layerSelector:
+    mediaType: "application/vnd.cncf.helm.chart.content.v1.tar+gzip"
+    operation: copy
+---
+apiVersion: helm.toolkit.fluxcd.io/v2
+kind: HelmRelease
+metadata:
+  name: opentelemetry-operator
+  namespace: ${workspaceNamespace}
+spec:
+  chartRef:
+    kind: OCIRepository
+    name: ghcr.io-opentelemetry-operator
+    namespace: ${workspaceNamespace}
+  interval: 15s
+  install:
+    remediation:
+      retries: 5
+    createNamespace: true
+  upgrade:
+    remediation:
+      retries: 5
```

```suggestion
  install:
    crds: CreateReplace
    remediation:
      retries: 30
    createNamespace: true
  upgrade:
    crds: CreateReplace
    remediation:
      retries: 30
```
Does this helm chart create any CRDs ? it might be handy to use `CreateReplace` to ensure upgrades are more predictable (default is `Create` i believe)

---

Yes, it does creates crd, added the recommended spec